packages/web/src/components/track/helpers.ts

```diff
@@ -79,3 +85,12 @@ export const getUserWithFallback = (user: User | null) => {
     }
   )
 }
+
+export const isDescendantElementOf = (
```

Comment:
https://github.com/AudiusProject/audius-client/blob/e42bf86d7e6ed496e0bb1ffbd03743c75083dc40/packages/web/src/utils/domUtils.ts#L1

mind putting this here or trying to re-use?